src/com/TTT/MiniMax.java

```diff
@@ -5,21 +5,94 @@
 
 public class MiniMax {
 
-    public int nextMove(Board board, String currentPlayerSymbol){
-        return -1;
+    private PlayerMarkers maxPlayer;
+
+    public int nextMove(Board board, PlayerMarkers maxPlayer){
+        this.maxPlayer = maxPlayer;
+
+        List<Integer> moves = board.availableMoves();
+        List<Integer> scores = new ArrayList<>();
+
+        for(int move : moves) {
+            Board clone = board.clone();
+            clone.placeMarker(move, maxPlayer.symbol());
+            scores.add(miniMax(maxPlayer, clone));
+        }
+
+        int bestScore = -200;
```

Comment:
It might be clearer to use constants for the magic numbers as they are used a few times.